frontend/tests/controllers/ContestUpdateTest.php

```diff
@@ -3,6 +3,36 @@
  * Description of ContestUpdateContest
  */
 class ContestUpdateTest extends \OmegaUp\Test\ControllerTestCase {
+    /**
+     * Check in DB for problem added to or updated in contest
+     *
+     * @param array $problemData
+     * @param array $contestData
+     * @param \OmegaUp\Request $r
+     */
+    public static function assertProblemAddedToOrUpdatedInContest(
+        $problemData,
+        $contestData,
+        $r
+    ) {
+        // Get problem and contest from DB
+        $problem = \OmegaUp\DAO\Problems::getByAlias(
+            $problemData['request']['problem_alias']
+        );
+        $contest = \OmegaUp\DAO\Contests::getByAlias(
+            $contestData['request']['alias']
+        );
+
+        // Get problem-contest and verify it
+        $problemset_problems = \OmegaUp\DAO\ProblemsetProblems::getByPK(
```

Comment:
Nit:
```suggestion
        $problemsetProblems = \OmegaUp\DAO\ProblemsetProblems::getByPK(
```
To maintain consistency with the other variable names